### PR TITLE
DEV: Added AT2L0 and test P_SATT

### PR DESCRIPTION
**Pull request by ghalym**
Interface tested between L-Arbiter and lfe-motion.

### CODE REVIEW

File: Arbiter/ArbiterPLC/POUs/P_SATT.TcPOU [hunk 1]
```diff
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="P_SATT" Id="{905bbaad-b3f0-420e-b1b5-0ee74487c823}" SpecialFunc="None">
+    <Declaration><![CDATA[PROGRAM P_SATT
+VAR
+	//All Sold attenuators requests are summerrized into this one output
```
Comment (klauer) [inline Arbiter/ArbiterPLC/POUs/P_SATT.TcPOU:6]:
```suggestion
	//All Solid attenuators requests are summarized into this one output
```

File: Arbiter/ArbiterPLC/POUs/P_SATT.TcPOU [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	{attribute 'TcLinkTo' := 'TIIB[plc-lfe-motion]^IO Outputs^SattArrayReq'}
+	q_SattArrayReq AT %Q* : ARRAY [1..PMPS_GVL.AUX_ATTENUATORS] OF ST_PMPS_Attenuator_IO; // Transmission request to ALL PLCs
+	//Per PLC current satt trans
+	{attribute 'TcLinkTo' := 'TIIB[plc-lfe-motion]^IO Inputs^SattArrayStatus'}
+	i_SattArrStatus_LFE AT %I* : ARRAY [1..PMPS_GVL.AUX_ATTENUATORS] OF ST_PMPS_Attenuator_IO; // Transmission request to KFE MOT PLC
```
Comment (klauer) [inline Arbiter/ArbiterPLC/POUs/P_SATT.TcPOU:11]:
```suggestion
	i_SattArrStatus_LFE AT %I* : ARRAY [1..PMPS_GVL.AUX_ATTENUATORS] OF ST_PMPS_Attenuator_IO; // Transmission status from LFE MOT PLC
```

File: Arbiter/ArbiterPLC/POUs/P_SATT.TcPOU [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	
+	// plc-tmo-mot interface diagnostics
```
Comment (klauer) [inline Arbiter/ArbiterPLC/POUs/P_SATT.TcPOU:13]:
```suggestion
	// plc-lfe-motion interface diagnostics
```

**Review by klauer — APPROVED** [review]
Looks good - just a couple comments to clean up.

File: Arbiter/ArbiterPLC/POUs/MAIN.TcPOU [hunk 4]
```diff
@@ -51,6 +51,7 @@ MachineSimulation();
 PMPS_Arbiter();
 A_SystemTime();
 
+P_SATT();
 
 fbLogHandler();]]></ST>
```
Comment (ZLLentz) [inline Arbiter/ArbiterPLC/POUs/MAIN.TcPOU:56]: Counterpart to my question on the other PR: in https://github.com/pcdshub/lcls-plc-kfe-arbiter/pull/9, we include a `Fast_Faults_Evaluate()` here. Is this not needed in LFE for some reason?

Comment (ZLLentz) [inline Arbiter/ArbiterPLC/POUs/MAIN.TcPOU:56]: Counterpart to my question on the other PR: in pcdshub/lcls-plc-kfe-arbiter#9, we include a `Fast_Faults_Evaluate()` here. Is this not needed in LFE for some reason?

**Review by ZLLentz — COMMENTED** [review]
My comments/questions on https://github.com/pcdshub/lcls-plc-kfe-arbiter/pull/9 also apply here

File: Arbiter/ArbiterPLC/POUs/MAIN.TcPOU (hunk 4 above)
Comment (ghalym) [reply Arbiter/ArbiterPLC/POUs/MAIN.TcPOU:56]: So, again part of the mystery, this issue didn't appear on this arbiter.

**Pull request merged**